ceremony/2022-10-03/repository/5.targets.json

```diff
@@ -0,0 +1,199 @@
+{
+	"signed": {
+		"_type": "targets",
+		"spec_version": "1.0",
+		"version": 5,
+		"expires": "2023-04-03T17:13:58Z",
+		"targets": {
+			"artifact.pub": {
+				"length": 177,
+				"hashes": {
+					"sha256": "59ebf97a9850aecec4bc39c1f5c1dc46e6490a6b5fd2a6cacdcac0c3a6fc4cbf",
+					"sha512": "308fd1d1d95d7f80aa33b837795251cc3e886792982275e062409e13e4e236ffc34d676682aa96fdc751414de99c864bf132dde71581fa651c6343905e3bf988"
+				},
+				"custom": {
+					"sigstore": {
+						"status": "Active",
+						"usage": "Unknown"
+					}
+				}
+			},
+			"ctfe.pub": {
+				"length": 177,
+				"hashes": {
+					"sha256": "7fcb94a5d0ed541260473b990b99a6c39864c1fb16f3f3e594a5a3cebbfe138a",
+					"sha512": "4b20747d1afe2544238ad38cc0cc3010921b177d60ac743767e0ef675b915489bd01a36606c0ff83c06448622d7160f0d866c83d20f0c0f44653dcc3f9aa0bd4"
+				},
+				"custom": {
+					"sigstore": {
+						"status": "Active",
+						"uri": "https://ctfe.sigstore.dev",
+						"usage": "CTFE"
+					}
+				}
```

Comment:
Should `ctlog2022.pub` also be in the top level targets?

Reply:
I decided against that (and I think you noted this earlier): it's a new target: so clients who are hardcoding paths like before can hardcode `ctfe/ctlog2022.pub`, and clients who are doing long-term stable changes of searching folders are also compatible.

Basically: no reason to backwards support the old format for a target that didn't exist.

Reply:
Sounds good! Just wanted to check.